Tests/Integration/AuthorizationServerTest.php

```diff
@@ -187,6 +202,37 @@ public function testValidClientCredentialsGrantWithScope(): void
         );
     }
 
+    public function testValidClientCredentialsGrantWithInheritedScope(): void
+    {
+        $request = $this->createAuthorizationRequest('quux_restricted_scopes:beer', [
```

Comment:
@spideyfusion Can we also use client IDs from constants? This makes it a lot easier to follow the test

Reply:
@alenpokos This will be resolved in a separate PR where we'll tackle test readability.